_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: newsletter
+title: "Newsletter #11 - Neovim 0.5"
```

Comment:
we need something catchy: `halfway there`, "the snail edition", "the LSP apocalypse" :p

Reply:
I like @bfredl's quote I used as a subtitle -- should I put this in the title as well?

Reply:
(Right now the title is chosen to mirror that of newsletter #10)

Reply:
I moved the subtitle to the title -- was that something like what you have in mind?

Reply:
I like the quote but it obfuscates the title